.prod/nginx.conf

```diff
@@ -11,6 +11,7 @@ server {
     }
 
     location / {
+        expires max;
```

Comment:
Are you sure sure? I dont have knowledge in nginx much but if you know what you are doing then its fine 🤔

Reply:
webpack gives each file in the bundle a unique name, so this should be safe